src/main/kotlin/com/zepben/evolve/database/sqlite/upgrade/changesets/ChangeSet39.kt

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2021 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+package com.zepben.evolve.database.sqlite.upgrade.changesets
+
+import com.zepben.evolve.database.sqlite.upgrade.ChangeSet
+
+internal fun changeSet39() = ChangeSet(39) {
+    listOf(
+        *`Add power transformer fields`,
+        *`Set power transformer fields based on diagram objects`
+    )
+}
+
+@Suppress("ObjectPropertyName")
+private val `Add power transformer fields` = arrayOf(
+    "ALTER TABLE power_transformers ADD construction_kind TEXT NOT NULL DEFAULT 'unknown'",
+    "ALTER TABLE power_transformers ADD function TEXT NOT NULL DEFAULT 'other'"
```

Comment:
how sweet is this its almost like we're using a real SQL database

Reply:
I think you could always add a column, it was removing and renaming that was improved